src/app/api/models/portfolio-submission.coffee

```diff
@@ -1,4 +1,7 @@
-angular.module("doubtfire.api.models.portfolio-submission", [
+_ = require('lodash')
+ngFileUpload = require('ng-file-upload')
```

Comment:
There are references to ngf-* directives in common/file-uploader which in turn seems to be referenced elsewhere throughout the app. possibly angular-file-upload is the deprecated one but it was ambiguous enough that I decided not to touch it